README.md

```
@@ -65,12 +94,320 @@ $meineVorlage = '
+// Umgebungsvariablen (alternative Lösung)
+$certPassword = $_ENV['CERT_PASSWORD'];
+$pdf->enableDigitalSignature('', $certPassword, ...);
+```
+
+**Best Practices für Zertifikate:**
+- **Produktive Zertifikate:** Nur von vertrauenswürdigen CAs verwenden
+- **Dateiberechtigungen:** 600 (nur Webserver lesbar) setzen
+- **Pfad-Sicherheit:** Zertifikate außerhalb des Web-Root speichern
+- **Ablaufmonitoring:** Rechtzeitige Erneuerung vor Ablauf
+- **Backup:** Sichere Aufbewahrung von Zertifikaten und Passwörtern
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)